mllib/src/main/scala/org/apache/spark/ml/feature/ChiSqSelector.scala

```diff
@@ -49,15 +49,13 @@ private[feature] trait ChiSqSelectorParams extends Params
    *
    * @group param
    */
-  @Since("1.6.0")
```

Comment:
In this case, I think it's OK to have Since annotations in the trait since it is private and should never be used beyond ChiSqSelector and ChiSqSelectorModel.  That seems pretty safe to me.